CMakeLists.txt

```diff
@@ -1,16 +1,11 @@
 cmake_minimum_required(VERSION 3.0.2)
 set(CMP0048 NEW)
 
-project(_tango LANGUAGES CXX
-        VERSION 1.0)
-        
-#need to define the version of the library
-set(MAJOR_VERSION "9")
-set(MINOR_VERSION "3")
-set(PATCH_VERSION "1")
+project(_tango LANGUAGES CXX)
+
+# Set PROJECT_VERSION and PACKAGE_VERSION from git tags
+include(cmake/project_version.cmake)
 
-#convenient versions
-set(LIBRARY_VERSION "${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}")
 if(NOT WIN32)
```

Comment:
`LIBRARY_VERSION` is not set anymore. Does this mean it was not being used before?
If this is the case, why not simply skip this since PyTango version is determined by tango/release.py?

Reply:
I git grepped for `LIBRARY_VERSION` but could not find a place where it is used. And there is already `PROJECT_VERSION` and ` PACKAGE_VERSION` defined by CMake so maybe it is not worth having an extra VERSION variable?

> why not simply skip this

Sorry, skip what exactly :-) ?

Reply:
> Sorry, skip what exactly :-) ?
I mean, `LIBRARY_VERSION` is not used in the PR made by @NexeyaSGara so why not just remove this line?

Reply:
But it is removed (in red), no?